app/models/link.rb

```diff
@@ -1,4 +1,6 @@
 class Link < ActiveRecord::Base
+  after_update :reset_time_and_status, if: :url_changed?
```

Comment:
We're adding more duplication with the LocalAuthority model.  I think this is OK, but I've added a Trello card to refactor this behaviour out -  https://trello.com/c/TiVpH2I4/331-consolidate-localauthority-homepage-url-and-link-url-behaviour

We could do this now?  Possibly need to talk it out in stand up?